src/c++/perf_analyzer/docs/examples/profile.py

```diff
@@ -76,12 +76,20 @@ def profile(args, input_data_file):
         f"perf_analyzer -m {args.model} -i grpc --async --streaming "
         f"--input-data={input_data_file} "
         "--profile-export-file=profile_export.json "
-        "--measurement-mode=count_windows "
-        "--measurement-request-count=10 "
-        "--stability-percentage=999"
     )
-    ret = subprocess.run(args=[command], shell=True)
-    ret.check_returncode()
+    if args.periodic_concurrency_range:
```

Comment:
Shouldn't the script take in multiple values for start, end, step, and request period? To sweep through them as RyanO indicated?

Reply:
Talked offline. Decided to avoid sweeping through them as the main purpose of the guide is to showcase different ways to profile LLM models using PA. We can mention that complete sweeping/search of the parameters will be done through model analyzer and it is on the way.